tests/plugins/autocomplete/manual/viewposition.html

```diff
@@ -0,0 +1,101 @@
+<style>
+	/* Swap margin with padding to prevent moving autocomplete panel outside editor. */
+	body {
+		margin-left: 0px;
```

Comment:
That's true. I will try to adjust caret position relative to the document body. I think it's an edge case where a user adds margin into body element but it could be probably fixed without huge work.